src/main/scala/it/pagopa/interop/purposeprocess/api/impl/PurposeApiServiceImpl.scala

```diff
@@ -489,6 +493,37 @@ final case class PurposeApiServiceImpl(
       onComplete(result) { clonePurposeResponse[Purpose](operationLabel)(clonePurpose200) }
     }
 
+  override def retrieveLatestRiskAnalysisConfiguration()(implicit
+    contexts: Seq[(String, String)],
+    toEntityMarshallerRiskAnalysisFormConfigResponse: ToEntityMarshaller[RiskAnalysisFormConfigResponse],
+    toEntityMarshallerProblem: ToEntityMarshaller[Problem]
+  ): Route = authorize(ADMIN_ROLE) {
+    val operationLabel = s"Retrieve latest risk analysis configuration"
+    logger.info(operationLabel)
+
+    val result: Future[RiskAnalysisFormConfigResponse] = for {
+      organizationId                   <- getOrganizationIdFutureUUID(contexts)
+      tenant                           <- tenantManagementService.getTenant(organizationId)
+      tenantKind                       <- tenant.kind.fold(
+        purposeVersionActivation.getTenantKindLoadingCertifiedAttributes(tenant.attributes, tenant.externalId)
+      )(Future.successful(_))
+      kindConfig                       <- riskAnalysisServiceSupplier
+        .get()
+        .riskAnalysisForms()
+        .get(tenantKind)
+        .toFuture(RiskAnalysisConfigForTenantKindNotFound(tenantKind))
+      (latest, riskAnalysisFormConfig) <- kindConfig.lastOption.toFuture(
```

Comment:
same for other PR, `lastOption` is not the correct approach on `Map`, we can use `maxByOption`